sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestAssetsHelper.cs

```diff
@@ -1,45 +1,55 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Azure.DigitalTwins.Core.Serialization;
 
 namespace Azure.DigitalTwins.Core.Tests
 {
     public static class TestAssetsHelper
     {
+        private static readonly string NewLine = Environment.NewLine;
+
+        // Remove all new line characters as they are differnt in differnt Operaing Systems.
+        // This will ensure that the recorded files always match the request in playback mode of tests.
+        private static string RemoveNewLine(string payload)
```

Comment:
Depending on which OS this code is running, the corresponding new line characters are removed